src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -24,25 +30,25 @@ export const styles = StyleSheet.create({
     flexDirection: 'column',
     height: height - 85
   },
+  flatlistContainer: {
+    paddingTop: 5,
+    paddingBottom: 20
+  },
   chatContainer: {
     display: 'flex',
     height: '100%'
   },
   inputContainer: {
     backgroundColor: colors.white,
-    position: 'absolute',
-    bottom: 0,
-    // height: 50,
-    left: 0,
-    right: 0,
     zIndex: 100,
     padding: 8,
-    paddingBottom: 16,
     borderTopColor: colors.lightgrey,
-    borderTopWidth: 1
+    borderTopWidth: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `inputContainer` style has been significantly changed. The position is no longer absolute and the bottom, left, right properties have been removed. This might affect the positioning of the input container on the screen. If this change was intentional and tested thoroughly across different screen sizes and devices, then it's fine. Otherwise, I would recommend testing it extensively or reverting to the previous style.

```diff
-    inputContainer: {
-      backgroundColor: colors.white,
-     position: 'absolute',
-     bottom: 0,
-     // height: 50,
-     left: 0,
-     right: 0,
-      zIndex: 100,
-      padding: 8,
-     paddingBottom: 16,
-      borderTopColor: colors.lightgrey,
-     borderTopWidth: 1
-    },
+    inputContainer: {
+       backgroundColor: colors.white,
+       zIndex: 100,
+       padding: 8,
+       borderTopColor: colors.lightgrey,
+      borderTopWidth: 1,
+      flexDirection: 'row',
+      alignItems: 'center',
+      justifyContent: 'center'
+     },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->